atom/browser/ui/accelerator_util.cc

```diff
@@ -109,6 +109,10 @@ bool StringToAccelerator(const std::string& description,
         modifiers |= ui::EF_SHIFT_DOWN;
     } else if (tokens[i] == "ctrl" || tokens[i] == "control") {
       modifiers |= ui::EF_CONTROL_DOWN;
+#if defined(OS_WIN)
```

Comment:
> To clarify, win will :
> 
> map the real keyboard win key on Windows/Linux.
> map cmd key on OS X.
> Is that correct?

Right.

> Maybe the shortcut should be named super instead of win ?

This seems like a better name. :+1: